R/batchmark.R

```diff
@@ -126,7 +127,13 @@ reduceBatchmarkResults = function(ids = NULL, keep.pred = TRUE, show.info = getM
       res = batchtools::reduceResultsList(tab[problem == prob & algorithm == algo], reg = reg)
       models = !is.null(res[[1L]]$model)
       lrn = data$learner[[algo]]
-      extract.this = getExtractor(lrn)
+      if (isTRUE(keep.extract)) {
```

Comment:
Why is this done here at the reducing stage and not in `getAlgoFun`. Now it just saves all the stuff to throw it away at the reduce stage, right?

Reply:
Good catch! :+1: